src/components/pages/Categories/Categories.vue

```diff
@@ -39,10 +51,32 @@ export default {
     errorMessage() {
       return this.$store.state.categories.errorMessage;
     },
+    doneMessage() {
+      return this.$store.state.categories.doneMessage;
+    },
   },
   created() {
     this.$store.dispatch('categories/getAllCategories');
   },
+  methods: {
+    clearMessage() {
+      this.$store.dispatch('categories/clearMessage');
+    },
+    resetMessage() {
+      this.$store.dispatch('categories/resetMessage');
+    },
+    updateValue(event) {
+      const { target } = event;
+      this.$store.dispatch('categories/updateValue', target.value);
```

Comment:
[nits]

一時変数は逆に可読性を低下させてしまいます。

```suggestion
      this.$store.dispatch('categories/updateValue', event.target.value);
```

Reply:
一時変数を削除しました。